src/server/bnetserver/Realms/WorldListener.cpp

```diff
@@ -65,7 +68,7 @@ void WorldListener::HandleOpen()
 {
     try
     {
-        _worldSocket->bind(std::string("tcp://*:") + std::to_string(_worldListenPort));
+        _worldSocket->bind(std::string("tcp://"+_bindIp+":") + std::to_string(_worldListenPort));
```

Comment:
Indeed it belongs to its own PR.
Also it doesnt have to be always local - bnetserver can be somewhere else.